#Distribute is a deprecated fork of the Setuptools project.
#Since the Setuptools 0.7 release, Setuptools and Distribute have merged and Distribute is no longer being maintained.
#http://pythonhosted.org/distribute/
#distribute legacy wrapper
#Package Documentation
#Distribute - legacy package
#This package is a simple compatibility layer that installs Setuptools 0.7+.
#https://pypi.python.org/pypi/distribute/0.7.3
#Since we have Setuptools > 0.7 in all (non-bare) Python version, we can remove if from the requirements.
#Alternatively I tried to install distribute as a part of the bundle, but the the sanity check always
#failed, and this way of solving the problem was easier for me.
#Aug 26th Aug 2016 by B. Hajgato (Free University Brussels - VUB)
diff -ru kiwisolver-0.1.3.org/kiwisolver.egg-info/requires.txt kiwisolver-0.1.3/kiwisolver.egg-info/requires.txt
--- kiwisolver-0.1.3.org/kiwisolver.egg-info/requires.txt	2014-07-12 14:07:20.000000000 +0200
+++ kiwisolver-0.1.3/kiwisolver.egg-info/requires.txt	2016-08-26 20:15:02.721088598 +0200
@@ -1 +0,0 @@
-distribute
\ No newline at end of file
diff -ru kiwisolver-0.1.3.org/setup.py kiwisolver-0.1.3/setup.py
--- kiwisolver-0.1.3.org/setup.py	2014-07-12 14:03:38.000000000 +0200
+++ kiwisolver-0.1.3/setup.py	2016-08-26 20:14:49.202095361 +0200
@@ -49,7 +49,6 @@
     url='https://github.com/nucleic/kiwi',
     description='A fast implementation of the Cassowary constraint solver',
     long_description=open('README.rst').read(),
-    install_requires=['distribute'],
     ext_modules=ext_modules,
     cmdclass={'build_ext': BuildExt},
 )
